[2024-05-29T08:57:24.796+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-29T08:57:24.816+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stored_procedures.load_results manual__2024-05-29T08:57:16.804132+00:00 [queued]>
[2024-05-29T08:57:24.823+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stored_procedures.load_results manual__2024-05-29T08:57:16.804132+00:00 [queued]>
[2024-05-29T08:57:24.824+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-29T08:57:24.835+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_results> on 2024-05-29 08:57:16.804132+00:00
[2024-05-29T08:57:24.842+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2846) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-29T08:57:24.844+0000] {standard_task_runner.py:63} INFO - Started process 2848 to run task
[2024-05-29T08:57:24.844+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'stored_procedures', 'load_results', 'manual__2024-05-29T08:57:16.804132+00:00', '--job-id', '195', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmpy3qxl8dh']
[2024-05-29T08:57:24.846+0000] {standard_task_runner.py:91} INFO - Job 195: Subtask load_results
[2024-05-29T08:57:24.883+0000] {task_command.py:426} INFO - Running <TaskInstance: stored_procedures.load_results manual__2024-05-29T08:57:16.804132+00:00 [running]> on host 29c09f03cc50
[2024-05-29T08:57:24.952+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Yamin' AIRFLOW_CTX_DAG_ID='stored_procedures' AIRFLOW_CTX_TASK_ID='load_results' AIRFLOW_CTX_EXECUTION_DATE='2024-05-29T08:57:16.804132+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-29T08:57:16.804132+00:00'
[2024-05-29T08:57:24.953+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-29T08:57:25.054+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-29T08:57:25.055+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-29T08:57:25.063+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=stored_procedures, task_id=load_results, execution_date=20240529T085716, start_date=20240529T085724, end_date=20240529T085725
[2024-05-29T08:57:25.098+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-29T08:57:25.122+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-29T08:57:25.124+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
